docs/guides/about/glossary.md

```diff
@@ -378,7 +381,13 @@ extent of the Australian intertidal zone at regular intervals of the observed ti
 
 For more information, see [DEA Intertidal Extents](https://www.dea.ga.gov.au/products/dea-intertidal-extents).
 
-{#jupyter-nb}      
+{#indexing}
+## Indexing
+
+The process of making data available (with a file location or URI) with associated metadata in a format
+that is accepted by the Datacube.
```

Comment:
Maybe something like:

"The process of registering a dataset with associated metadata (in a folder or at a URI) to an OpenDataCube instance so that it is searchable and accessible through the datacube API."

?